breaking-front/src/pages/PostWrite/units/PostWriteSearchLocation.js

```diff
@@ -0,0 +1,251 @@
+        roadAddressName: adresses[0].road_address?.address_name,
+        addressName: adresses[0].address.address_name,
+      }));
+      setIsCustomMarker(true);
+    }
+  };
+
+  const SetCustomMarker = (lat, lng) => {
+    // 클릭위치에 마커가 찍어지도록 state를 변경
+    setMarkerInformation({
+      lat: lat,
+      lng: lng,
+    });
+    geocoder.coord2Address(lng, lat, coord2AdressCallback);
+  };
+
+  const toggleModal = () => {
+    setIsModalOpen((pre) => !pre);
+  };
+
+  const handleSearchInput = (event) => {
+    setSearchContent(event.target.value);
+  };
+
+  const SearchMap = (event) => {
+    event.preventDefault();
+    const ps = new kakao.maps.services.Places();
+
+    ps.keywordSearch(searchContent, (data, status) => {
+      if (status === kakao.maps.services.Status.OK) {
+        // 검색된 장소 위치를 기준으로 지도 범위를 재설정하기위해
+        // LatLngBounds 객체에 좌표를 추가합니다
+        const bounds = new kakao.maps.LatLngBounds();
+        setSearchResult(data);
+        for (var i = 0; i < data.length; i++) {
+          bounds.extend(new kakao.maps.LatLng(data[i].y, data[i].x));
+        }
+        // 검색된 장소 위치를 기준으로 지도 범위를 재설정합니다
+        map.setBounds(bounds);
+      }
+    });
+  };
+
+  useEffect(() => {
+    map && map.relayout();
+    // modal과 같이 display의 값이 바뀌는 곳에서는  map.relayout() 가 필요
+  }, [isModalOpen, map]);
+  return (
+    <>
+      <Style.PostWriteTitle>
+        위치
+        <Style.FindLocationLayout onClick={GetCurrentPosition}>
+          <LocationIcon />
+          <Style.FindLocationMessage>현재 위치 찾기</Style.FindLocationMessage>
+        </Style.FindLocationLayout>
+      </Style.PostWriteTitle>
+
+      <Style.LocationInput
+        type="text"
+        placeholder="주소를 입력하세요"
+        onClick={toggleModal}
+        value={locationInputValue}
+        readOnly
+      />
+      <PostWriteModal
+        title="위치 찾기"
+        isOpen={isModalOpen}
+        closeClick={toggleModal}
+      >
+        <Map
+          center={mapCenterPosition}
+          isPanto={true}
+          style={{
+            // 지도의 크기
+            width: '100%',
+            height: '640px',
+            borderRadius: '0px 0px 10px 10px',
+          }}
+          level={8} // 지도의 확대 레벨
+          onCreate={setMap} //map 객체를 받아옴
+          onClick={(_t, mouseEvent) => {
+            // 맵을 클릭시에 실행되는 eventHandler
+            SetCustomMarker(
+              mouseEvent.latLng.getLat(),
+              mouseEvent.latLng.getLng()
+            );
+          }}
+        >
+          {/*맵을 클릭했을때 나오는 마커 */}
+          {isCustomMarker && (
+            <MapMarker
+              position={{
+                lat: markerInformation.lat,
+                lng: markerInformation.lng,
+              }}
+              clickable={true}
+              onClick={LocationSubmit}
+            >
+              <Style.SearchMarker>
+                <Style.PlaceName>
+                  {markerInformation?.addressName}
+                </Style.PlaceName>
+              </Style.SearchMarker>
```

Comment:
```suggestion
              <Style.SearchMarker>
                  {markerInformation?.addressName}
              </Style.SearchMarker>
```
PlaceName 없애고 SearchMarker에 `font-size:12px;` 스타일 추가해도 되지 않을까요?